manager/manager/views/ui.py

```diff
@@ -504,6 +504,45 @@ def address_delete(request, address_id=None):
     return redirect("address_list")
 
 
+# order list hlper function -
+def apply_orders_sorting(queryset, sort_field, sort_dir):
+    """Apply sorting to orders queryset."""
+    order_prefix = '' if sort_dir == 'asc' else '-'
+    if sort_field == 'min_id':
+        return queryset.order_by(f"{order_prefix}id")
+    elif sort_field == 'created_by':
+        return queryset.order_by(f"{order_prefix}created_by__user__first_name")
+    elif sort_field == 'created_on' or sort_field == 'status':
+        return queryset.order_by(f"{order_prefix}{sort_field}")
+    elif sort_field in ('config_name', 'country'):
+        orders_list = list(queryset)
+        if sort_field == 'config_name':
+            orders_list.sort(
+                key=lambda order: str(order.data.get('config', {}).get('name', '')).lower() if order.data else '',
+                reverse=(sort_dir == 'desc')
+            )
+        else:
+            orders_list.sort(
+                key=lambda order: str((order.data.get('recipient', {}) or {}).get('country', '')).lower() if order.data else '',
+                reverse=(sort_dir == 'desc')
+            )
+        if orders_list:
+            from django.db.models import Case, When, Value, IntegerField
+            preserved_order = [order.id for order in orders_list]
```

Comment:
There's no need to store that in a variable. It's only useful to build `preserved_order_cases`